cypress/support/commands.js

```diff
@@ -65,25 +65,22 @@ Cypress.Commands.add('mountWithContext', (Component, options = {}, props) => {
 Cypress.Commands.add(
   'mockWindowChrome',
   ({ userPermissions } = { userPermissions: ['*:*:*'] }) => {
-    cy.window().then(
-      // one of the fec dependencies talks to window.insights.chrome
-      (window) => {
-        window.chrome = {
-          getApp: () => 'inventory',
-        };
-        window.insights = {
-          chrome: {
-            getUserPermissions: () => userPermissions,
-            auth: {
-              getUser: () => {
-                return Promise.resolve({});
-              },
-            },
-            getApp: () => 'inventory',
+    window.chrome = {
```

Comment:
Would mocking all this with the `useChrome` override not make the use of the `window` object obsolete?

Reply:
This is true, it seems this property modification is already obsolete. I will try now to remove it

Reply:
done in the next commit

Reply:
Also, I didn't remove the window.insights mock because some parts of the code (but not Inventory's) still refers to window.insights.chrome object for some reason.
```
(uncaught exception)Error: Objects are not valid as a React child (found: TypeError: Cannot read properties of undefined (reading 'chrome')). If you meant to render a collection of children, use an array instead.
```

Reply:
I think this should be remnants of the chrome older version which embedded the chrome into the window heavily. Maybe if we find what packages are still using that old version and update them, we can remove this final bit as well.

Reply:
Yeah, we can. But not sure it will contribute much to the issue with the tests I am trying to solve currently. I will keep you note in mind and fix in a separate PR.

Also merging this. Looks like at least 2 people took a look and there is almost no way to verify this PR, so let's see if it heals CI on master.